include/openssl/ssl.h.in

```diff
@@ -2029,6 +2029,7 @@ struct evp_pkey_st *SSL_get_privatekey(const SSL *ssl);
 
 __owur X509 *SSL_CTX_get0_certificate(const SSL_CTX *ctx);
 __owur EVP_PKEY *SSL_CTX_get0_privatekey(const SSL_CTX *ctx);
+__owur int SSL_CTX_remove_cert_and_key(SSL_CTX *ctx);
```

Comment:
You are adding a new public API.  You  need to do `make update` to regenerate some things.